### ISSUE TITLE
Support recursive populate

### ISSUE THREAD

**Issue by vjpr** [Arro/airtable-json#4]
If a linked record links to another record, allow populating it.

**Comment by Arro** [comment]
I've thought about this one.  Could get messy though.  Would be a useful feature, no doubt.

**Comment by richarddavenport** [comment]
@vjpr @Arro I was just about to implement this... I think it could be straight forward, just need to be recursive. @Arro are you up for a PR?

**Comment by Arro** [comment]
> @vjpr @Arro I was just about to implement this... I think it could be straight forward, just need to be recursive. @Arro are you up for a PR?

100% yes.  Go for it.

**Comment by richarddavenport** [comment]
@Arro Do you know anything about the rate limits on airtable? I'm wondering if we'll run into that?

**Comment by Arro** [comment]
Not really, but I've sought to hit their API as little as possible (per function call) for those reasons.

**Comment by nathanaelphilip** [comment]
@richarddavenport were you able to implement this?

**Comment by richarddavenport** [comment]
@nathanaelphilip I ended up just using the API directly. I'd have to take another look to see where I was at with this.

**Comment by nathanaelphilip** [comment]
@richarddavenport no problem! thanks for the response. I’ll probably end up doing the same to help prevent overloading the API

**Comment by richarddavenport** [comment]
@nathanaelphilip I have a feeling that if you do make recursive calls it will be the same amount of api calls if you don't do it recursive. I think you have to make an api call per "table".
